src/Graph.h

```diff
@@ -116,26 +116,62 @@ class Graph
     unsigned int getSize();
 
     /**
+     * Dijkstra method
      * findPathLengths() will get all the path lengths with a certain node from the center.
```

Comment:
what do you mean with a certain node?  can you clarify that?

Reply:
changed description to be more accurate